2017-7-4 17:17:59--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;An attempt to attach an auto-named database for file c:\users\dxn\documents\visual studio 2013\Projects\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB1.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-07-22 21:46:40--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 50 - 发生了 Local Database Runtime 错误。无法创建自动实例。有关错误详细信息，请参阅 Windows 应用程序事件日志。
)
SQL:GetSchema 
CommandType:Text
Parameters:

2017-07-22 21:49:33--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 50 - 发生了 Local Database Runtime 错误。无法创建自动实例。有关错误详细信息，请参阅 Windows 应用程序事件日志。
)
SQL:GetSchema 
CommandType:Text
Parameters:

2017-07-22 21:50:15--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 50 - 发生了 Local Database Runtime 错误。无法创建自动实例。有关错误详细信息，请参阅 Windows 应用程序事件日志。
)
SQL:GetSchema 
CommandType:Text
Parameters:

2017-07-22 21:53:18--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 50 - 发生了 Local Database Runtime 错误。无法创建自动实例。有关错误详细信息，请参阅 Windows 应用程序事件日志。
)
SQL:GetSchema 
CommandType:Text
Parameters:

2017-07-22 21:54:53--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 50 - 发生了 Local Database Runtime 错误。无法创建自动实例。有关错误详细信息，请参阅 Windows 应用程序事件日志。
)
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-24 11:21:01--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB_VS2017.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-24 11:21:21--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB_VS2017.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-24 11:25:50--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB_VS2017.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-24 12:06:57--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-24 13:53:49--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-24 13:54:17--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-24 13:55:08--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-24 13:55:20--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-24 16:01:52--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-25 13:34:09--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-25 13:35:23--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-25 13:35:46--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-26 17:00:45--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-26 17:01:11--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-07-29 11:12:31--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 50 - 发生了 Local Database Runtime 错误。无法创建自动实例。有关错误详细信息，请参阅 Windows 应用程序事件日志。
)
SQL:GetSchema 
CommandType:Text
Parameters:

2017-07-29 11:12:31--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 50 - 发生了 Local Database Runtime 错误。无法创建自动实例。有关错误详细信息，请参阅 Windows 应用程序事件日志。
)
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-31 17:16:25--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB_VS2017.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-31 17:32:04--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-31 17:32:27--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-31 17:42:41--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-31 17:43:02--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-31 18:24:08--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;连接超时时间已到。在尝试使用预登录握手确认时超过了此超时时间。这可能是因为预登录握手失败或服务器未能及时响应。  尝试连接到此服务器时花费的持续时间是 - [Pre-Login] initialization=11；handshake=0；
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-31 18:26:18--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-31 18:27:59--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;连接超时时间已到。在尝试使用预登录握手确认时超过了此超时时间。这可能是因为预登录握手失败或服务器未能及时响应。  尝试连接到此服务器时花费的持续时间是 - [Pre-Login] initialization=12；handshake=0；
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-31 18:31:41--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;连接超时时间已到。在尝试使用预登录握手确认时超过了此超时时间。这可能是因为预登录握手失败或服务器未能及时响应。  尝试连接到此服务器时花费的持续时间是 - [Pre-Login] initialization=13；handshake=0；
SQL:GetSchema 
CommandType:Text
Parameters:

2017-7-31 18:32:27--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;连接超时时间已到。在尝试使用预登录握手确认时超过了此超时时间。这可能是因为预登录握手失败或服务器未能及时响应。  尝试连接到此服务器时花费的持续时间是 - [Pre-Login] initialization=59；handshake=0；
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 9:20:06--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;连接超时时间已到。在尝试使用预登录握手确认时超过了此超时时间。这可能是因为预登录握手失败或服务器未能及时响应。  尝试连接到此服务器时花费的持续时间是 - [Pre-Login] initialization=1209；handshake=0；
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 9:33:20--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;连接超时时间已到。在尝试使用预登录握手确认时超过了此超时时间。这可能是因为预登录握手失败或服务器未能及时响应。  尝试连接到此服务器时花费的持续时间是 - [Pre-Login] initialization=28；handshake=0；
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 9:52:07--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;连接超时时间已到。在尝试使用预登录握手确认时超过了此超时时间。这可能是因为预登录握手失败或服务器未能及时响应。  尝试连接到此服务器时花费的持续时间是 - [Pre-Login] initialization=1087；handshake=0；
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 9:52:37--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 9:53:13--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 9:53:50--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 9:54:37--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;连接超时时间已到。在尝试使用预登录握手确认时超过了此超时时间。这可能是因为预登录握手失败或服务器未能及时响应。  尝试连接到此服务器时花费的持续时间是 - [Pre-Login] initialization=1209；handshake=0；
SQL:GetSchema 
CommandType:Text
Parameters:
===============

2017-8-1 10:33:23--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 10:34:22--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 10:34:55--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 10:50:27--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 11:37:46--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 11:38:55--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 13:01:38--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
Cannot create file 'F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf' because it already exists. Change the file path or the file name, and retry the operation.
Could not open new database 'F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
File activation failure. The physical file name "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf" may be incorrect.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 13:02:19--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 13:02:31--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 13:08:48--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;连接超时时间已到。在尝试使用预登录握手确认时超过了此超时时间。这可能是因为预登录握手失败或服务器未能及时响应。  尝试连接到此服务器时花费的持续时间是 - [Pre-Login] initialization=14；handshake=0；
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 13:14:52--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 13:16:52--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 13:18:12--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-1 13:52:09--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-2 10:06:00--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-2 10:06:02--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-2 10:19:19--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-2 10:52:32--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-4 11:25:03--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-4 11:29:01--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-4 11:29:25--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-4 13:30:52--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-4 13:47:20--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
Cannot create file 'F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf' because it already exists. Change the file path or the file name, and retry the operation.
Could not open new database 'F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
File activation failure. The physical file name "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf" may be incorrect.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-4 13:51:19--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
Cannot create file 'F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf' because it already exists. Change the file path or the file name, and retry the operation.
Could not open new database 'F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
File activation failure. The physical file name "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf" may be incorrect.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-7 8:57:28--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-7 8:58:25--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-7 10:37:05--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-7 10:52:04--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
Cannot create file 'F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf' because it already exists. Change the file path or the file name, and retry the operation.
Could not open new database 'F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
File activation failure. The physical file name "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf" may be incorrect.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-7 10:52:19--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
Cannot create file 'F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf' because it already exists. Change the file path or the file name, and retry the operation.
Could not open new database 'F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
File activation failure. The physical file name "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf" may be incorrect.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-7 10:52:57--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
Cannot create file 'F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf' because it already exists. Change the file path or the file name, and retry the operation.
Could not open new database 'F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
File activation failure. The physical file name "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf" may be incorrect.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-7 10:54:36--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
Cannot create file 'F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf' because it already exists. Change the file path or the file name, and retry the operation.
Could not open new database 'F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
File activation failure. The physical file name "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf" may be incorrect.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-7 10:55:29--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Unable to open the physical file "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf". Operating system error 32: "32(另一个程序正在使用此文件，进程无法访问。)".
Cannot create file 'F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf' because it already exists. Change the file path or the file name, and retry the operation.
Could not open new database 'F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
File activation failure. The physical file name "F:\Git\PWMIS.OAuth2\PWMIS.OAuth2.AuthorizationCenter\App_Data\OAuth2DB_log.ldf" may be incorrect.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-8 15:13:29--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

2017-8-8 15:13:30--identity_repository_error:PDF.NET AdoHelper Query Error：
DataBase ErrorMessage:;Cannot open database "F:\GIT\PWMIS.OAUTH2\PWMIS.OAUTH2.AUTHORIZATIONCENTER\APP_DATA\OAUTH2DB.MDF" requested by the login. The login failed.
Login failed for user 'DXN-DTH-PC\dxn'.
SQL:GetSchema 
CommandType:Text
Parameters:

